solara/lab/components/theming.vue

```diff
@@ -0,0 +1,70 @@
+<template>
+    <v-btn
+        icon
+        @click="countClicks"
+        >
+        <v-icon
+            :color="theme_effective ? 'primary' : null">
+            {{ this.clicks === 1 ? this.on_icon : this.clicks === 2 ? this.off_icon : this.auto_icon }}
+        </v-icon>
+    </v-btn>
+</template>
+<script>
+module.exports = {
+    mounted() {
+        if (theme.variant_user_selectable) {
+            if (localStorage.getItem('theme.variant')) {
```

Comment:
```suggestion
            if (localStorage.getItem(':solara:theme.variant')) {
```
Shall we prefix it with something like this? Other prefix ideas are welcome :)
This reduces the chance of future name collisions with user chosen names